project/ws/app/src/lib/routes/profile-v2/components/designation-request/designation-request.component.html

```diff
@@ -0,0 +1,11 @@
+<div class="flex flex-col gap-4">
+    <div class="text-base font-bold text-center">Request to Add Designation</div>
+    <div class="flex flex-col gap-1">
+        <div class="text-sm font-bold required">Enter designation</div>
```

Comment:
please mention jira ticket ids in pr title

Reply:
Done